Documentation/Elastics/ElasticSystem.md

```diff
@@ -0,0 +1,76 @@
+# Elastic system (experimental)
+
+![Elastic System](../Images/Elastics/Elastics_Main1.gif)
+
+MRTK comes with an elastic simulation system that includes a wide variety of extensible and flexible subclasses, offering bindings for 4-dimensional quaternion springs, 3-dimensional volume springs and simple linear spring systems.
+
+Currently the following MRTK components supporting the [elastics manager](xref:Microsoft.MixedReality.Toolkit.Experimental.Physics.ElasticsManager) can leverage elastics functionality:
+
+- [Bounds control](../README_BoundsControl.md)
+- [Object manipulator](../README_ObjectManipulator.md)
+
+## Elastics manager
+![Elastic System2](../Images/Elastics/Elastics_Main.gif)
+
+The elastics manager takes over processing passed transforms and feeding them into the elastics system. 
+
+Enabling elastics for custom components can be achieved by two steps:
+1. Calling the Initialize method on manipulation start, updating the system with the current host transform.
+1. Querying ApplyHostTransform whenever a elastics calculation should be performed on the updated target transform.
+
+Note that elastics will continue simulating once manipulation ends (through the elastics manager update loop). To block elastics auto update [EnableElasticsUpdate](xref:Microsoft.MixedReality.Toolkit.Experimental.Physics.ElasticsManager.EnableElasticsUpdate) can be set to false.
+
+
+The elastics manager component when added to a game object won't have elastics enabled for any transforms type per default.
+The field `Manipulation types using elastic feedback` needs to be enabled for specific transform types to create elastics configuration and extents for the selected type.
+
+### Elastics configurations
+
+Similar to [bounds control configurations](../README_BoundsControl.md#configuration-objects), elastic manager comes with a set of configuration objects that can be stored as scriptable objects and shared between different instances or prefabs. Configurations can be shared and linked either as individual scriptable asset files or nested scriptable assets inside of prefabs. Further configurations can also be defined directly on the instance without linking to an external or nested scriptable asset.
+
+The elastics manager inspector will indicate whether a configuration is shared or inlined as part of the current instance by showing a message in the property inspector. In addition shared instances won't be editable directly in the elastics manager property window itself, but instead the asset it's linking to has to be directly modfied to avoid any accidental changes on shared configurations.
```

Comment:
```suggestion
The elastics manager inspector will indicate whether a configuration is shared or inlined as part of the current instance by showing a message in the property inspector. In addition, shared instances won't be editable directly in the elastics manager property window itself, but instead the asset it's linking to has to be directly modfied to avoid any accidental changes on shared configurations.
```